www/util.php

```diff
@@ -2915,7 +2915,7 @@ function close_user_acct($db, $uid, $stat, &$progress)
 
     // unlock tables, if we haven't already
     if ($tableLocks)
-        mysql_query("unlock tables");
+        mysql_query("unlock tables", $db);

```

Comment:
This line is supposed to be unreachable; it would only be reached if there were errors after locking that prevented unlocking.

Reply:
I found a way to test it by adding a tag as test@ifdb.org then deliberately adding a SQL syntax bug on line 2799, and trying to delete test@ifdb.org. In main, the deletion fails and the result page is blank. In your branch, it works (displaying an error message) as intended.